client/src/theme.js

```diff
@@ -73,6 +74,34 @@ const theme = createTheme({
         },
       },
     },
+    MuiButton: {
+      variants: [
+        {
+          props: { variant: "transparent" },
+          style: {
+            fontSize: ".9rem",
+            textTransform: "none",
+            background: "none",
+            borderRadius: "20px",
+          },
+        },
+        {
+          props: { variant: "rounded", color: "primary" },
+          style: {
+            textTransform: "none",
+            fontSize: ".9rem",
+            borderRadius: "20px",
+            backgroundColor: "#F6A794",
+            color: "#021944",
+            "&:hover": {
+              backgroundColor: "#F6A794",
```

Comment:
are these theme colors we should pull from the palette?

Reply:
I've been trying to figure this out. The problem is that the palette is defined in theme and so is this part of the theme so it cannot refer to itself to get the colours if it is part of the definition of theme itself. Another thing we can do is pull this part out of the theme and define it elsewhere while referring to the theme but then I would have to redefine the same thing again to add more to it and I feel like it will get messy and disorganized

Reply:
Oh yeah so we could also add another named color map to the theme file if we want, like a hash map

```js
const COLORS = {
  taro: '#123123',
  redBean: '#123123',
}
```

as well, and then in the theme definitions, we can extend that

```js
export const theme = createTheme({
  palette: {
    primary: {
      main: COLORS.taro
    }
  }
})
```

Reply:
should we make this another ticket or should I directly do it directly here? (rather than everyone making changes to their own ticket to adapt to this change we can make all the changes in a new ticket)

Reply:
Yeah, I think this would help with organization and matching with the design. One thing I'm worried about is that there are a lot of colours that are used in the figma and most of them are unnamed or used only once or twice. It might difficult to match a colour to a name in the theme.

What do you think?

Reply:
Yeah, I understand how this would benefit in terms of organization. One problem I notice is that there are a lot of colours, and most colors in the figma are unnamed and are only used once or twice. If we do it this way, we will be basically giving every colour two names...and most will not be repeated.

I just thought of another way we can try to do it that is similar to your method...maybe we can define all the colours in a separate object and then add it to the theme like so:

```
const COLORS = {
   brown: {
      ...
   },
   gray: {
      ...
   },
};

const theme = createTheme({
   palette: {
       ...COLORS
   },
   components: {
       MuiButton: {
           color: COLORS.brown[100],
       }
   },
});
```

Reply:
If there are too many diff colors in the design, it might be an opportunity to ask design and potentially correct an unintentional color variation, if design wants 10 diff shades of brown though, we can do it, too

But never hurts to ask, designs aren't the final say, as devs I think it's important to pushback a little too, question when things seem weird

Reply:
I'll ask the design 👍